README.md

```diff
@@ -1,34 +1,111 @@
-# Tutorial on Trixi.jl at ICOSAHOM 2021
+# Trixi.jl: High-Order Numerical Simulations in Julia
```

Comment:
Something with conservation laws or hyperbolic PDEs? Otherwise, I fear it's just too general and we will get beef with elliptic/parabolic/ODE/... people

Reply:
Yes, I feared this would come up :-) Unfortunately, I didn't come up with anything better that is more precise, not boring, and still succinct. I am very much open for suggestions!

Reply:
- Trixi.jl: High-Order Numerical Simulations of Conservation Laws in Julia
- Trixi.jl: High-Order Numerical Simulations of Hyperbolic PDEs in Julia

Reply:
- Join the hype(erbolic PDE world) with Trixi.jl: High-Order Numerical Simulations in Julia

Reply:
I used the second version you suggested due to its (relative) brevity. Done in b19a717